ddtrace/profiling/collector/_traceback.pyx

```diff
@@ -12,7 +12,8 @@ cpdef traceback_to_frames(traceback, max_nframes):
         if nframes < max_nframes:
             frame = tb.tb_frame
             code = frame.f_code
-            frames.insert(0, (code.co_filename, frame.f_lineno, code.co_name))
+            lineno = 0 if frame.f_lineno is None else frame.f_lineno
```

Comment:
`frame.f_lineno or 0`?

Reply:
I do prefer to be explicit around None here, not just using any "false" value.